### ISSUE TITLE
Warnings are always suppressed, can't find a setting

### ISSUE THREAD

**Issue by dickwall** [lihaoyi/Ammonite#296]
I was using ammonite for some training and noticed that the compiler warnings are all suppressed. While I would typically agree that warnings in a REPL are often more noise than value, during the training we were specifically trying to call out unchecked warnings and other compiler hints so I dropped back to the plain old REPL for those. It would be great to have a setting to re-enable warnings for demo purposes or other correctness checking.

**Comment by lihaoyi** [comment]
This was fixed in 0.5.3
```scala
@ // Enabling warnings (which are disabled by default)

@ List(1) match { case _: List[Double] => 2 }
res7: Int = 2

@ compiler.settings.nowarnings.value = false

@ List(1) match { case _: List[Double] => 2 }
warning: $fruitlessTypeTestWarningMessageBlahBlahBlah
```
@dickwall

**Comment by ryan-richt** [comment]
I also see this regression on version 2.5.2
